gtdynamics.i

```diff
@@ -30,10 +30,20 @@ class JointMeasurementFactor : gtsam::NonlinearFactor {
 };
 
 #include <gtdynamics/factors/PoseFactor.h>
-gtsam::NonlinearFactor PoseFactor(gtsam::Key wTp_key, gtsam::Key wTc_key,
-                                  gtsam::Key q_key,
-                                  const gtsam::noiseModel::Base *cost_model,
-                                  const gtdynamics::Joint *joint);
+class PoseFactor : gtsam::NonlinearFactor {
```

Comment:
Please do not change this function. Leave it as Yetong did it. I can't believe how much energy I'm spending on this.

Reply:
This is what you get for not discussing these things with us in advance. :slightly_smiling_face:

Reply:
I mean, you did introduce a bug in the codebase, so it becomes your responsibility. :sweat_smile:  I can't believe how much time and energy __I__ am spending on this when I should be working on state estimation.